processing/src/main/java/io/druid/segment/incremental/IncrementalIndexAdapter.java

```diff
@@ -269,32 +266,15 @@ public ColumnCapabilities getCapabilities(String column)
   }
 
   @Override
-  public BitmapIndexSeeker getBitmapIndexSeeker(String dimension)
+  public IndexedInts getBitmapIndex(String dimension, int dictId)
   {
-    final Map<String, MutableBitmap> dimInverted = invertedIndexes.get(dimension);
-    if (dimInverted == null) {
-      return new EmptyBitmapIndexSeeker();
+    if (dictId >= 0) {
+      final Indexed<String> dimValues = getDimValueLookup(dimension);
+      String value = Strings.nullToEmpty(dimValues.get(dictId));
```

Comment:
I think this warrants a comment, given that it took me a while to track down the reason for this.